packages/address/src/AddressStyles.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components'
+
+export const AddressStyle = styled.div`
+  position: relative;
+  display: inline-block;
+  font-weight: 500;
+  margin: 0 6px;
+
+  &:empty {
+    display: none;
+  }
```

Comment:
These styles were in the IdenticonBadge for hiding empty address to avoid extra indention when address is not provided. This no longer works because the component always has full address inside the HTML: 

![image](https://user-images.githubusercontent.com/1422379/128463444-56c1aade-4016-45a8-a79d-c49b774d66f5.png)

And it seems better to fix it in the IdenticonBadge instead of the Address.